tezos/messages/Limits.md

```diff
@@ -0,0 +1,440 @@
+# P2P Messages
```

Comment:
@akoptelov  @jurajselep 
I saw chat on slack about README.md and include this Limits.md there, 
I think Juraj is right, why not to link it from tezos/messages/README.md or just rename Limits.md to README.md ?

now when you open sorces for tezos/messages it shows nothing:
https://github.com/akoptelov/tezedge/tree/encoding-doc-gen/TE-477/tezos/messages